src/move/data/dataloaders.py

```diff
@@ -10,9 +10,23 @@
 from move.core.typing import BoolArray, FloatArray
 
 
-class MOVEDataset(TensorDataset):
-    "Characterizes a dataset for PyTorch"
+class MOVEDataset(TensorDataset):    
+    """
+    Characterizes a dataset for PyTorch
+
+    Attributes:
+        TensorDataset (class): class to represent data as list of tensors
+        
+    Args:
+        cat_all (torch.Tensor, optional): categorical input matrix (N_patients, N_variables x N_max-classes. Defaults to None.
```

Comment:
No need to specify the types of arguments if they are already annotated in the class/function signature.

Reply:
Fixed it: a48aaa0